test/e2e/flowaggregator_test.go

```diff
@@ -936,12 +930,7 @@ func checkAntctlGetFlowRecordsJson(t *testing.T, data *TestData, podName string,
 	// run antctl command on flow aggregator to get flow records
 	var command []string
 	args := []string{"get", "flowrecords", "-o", "json", "--srcip", srcIP, "--srcport", srcPort}
-	if testOptions.enableCoverage {
-		antctlCovArgs := []string{"antctl-coverage"}
-		command = append(antctlCovArgs, args...)
-	} else {
-		command = append([]string{"antctl"}, args...)
-	}
+	command = append([]string{"antctl"}, args...)
```

Comment:
nit: you can do the same thing as above: use `command := ...` and remove the `var command []string` statement